src/routes/company/+page.svelte

```diff
@@ -29,560 +71,711 @@
-                        <p class="web-description web-u-max-width-480 u-margin-inline-auto">
-                            Appwrite is proudly backed by some of the top investors in the industry.
-                        </p>
-                    </div>
-                    <ul class="web-investor-cards web-u-margin-block-start-64">
-                        <li>
-                            <a
-                                href="https://www.bvp.com/"
-                                class="web-card"
-                                target="_blank"
-                                rel="noopener noreferrer"
-                            >
-                                <div class="web-card is-full-color" style="--card-padding:1rem;">
-                                    <img
-                                        src="/images/investors/light/bessemer.svg"
-                                        alt=""
-                                        height="63"
-                                    />
+                    <div class="web-timeline-content">
+                        <ol class="web-timeline-content-list">
+                            <li class="web-timeline-content-item">
+                                <div class="web-timeline-content-item-top web-grid-1-1"
+                                     style="--grid-1-1-gap:2.5rem; --grid-1-1-gap-desktop:15rem;">
+                                    <div class="u-flex-vertical u-gap-16">
+                                        <h3 class="web-title web-u-color-text-primary">
+                                            Ship faster
+                                        </h3>
+                                        <p class="web-main-body-500">
+                                            Utilizing Appwrite as your backend, you significantly cut down the time and resources spent on building a backend infrastructure from scratch. This means you can get to quicker iterations and faster time-to-market, directly addressing the challenge of finding product-market fit sooner.
+                                        </p>
+                                    </div>
+                                    <div >
+                                        <div class="web-card is-white u-flex-vertical web-u-gap-20">
+                                            <p class="aw-sub-body-500">The integrated user authentication and the ease of creating data structures have undoubtedly saved us several weeks' worth of time.</p>
+                                            <div class="web-user-box">
+                                                <img class="web-user-box-image" src="/images/community/avatars/kap.png" alt="Avatar of Kap.ts">
+                                                <div class="web-user-box-name web-sub-body-500">Marius Bolik</div>
+                                                <div class="web-user-box-username web-sub-body-400">CTO // mySHOEFITTER</div>
```

Comment:
```suggestion
                                                <div class="web-user-box-username web-sub-body-400">CTO @ mySHOEFITTER</div>
```
![Screenshot 2024-04-15 at 11 44 05 PM](https://github.com/appwrite/website/assets/20852629/17ed5ef9-3f29-43ae-93eb-193b3ca5cac3)

I believe its more common and relevant to use the `@` instead of `//` . Please check this option with the design team as well

Reply:
Design team prefer `//` as @ can be confused with social handle